x-pack/legacy/plugins/siem/public/pages/network/ip_details/utils.ts

```diff
@@ -36,7 +36,9 @@ export const getBreadcrumbs = (params: NetworkRouteSpyState, search: string[]):
       ...breadcrumb,
       {
         text: decodeIpv6(params.detailName),
-        href: `${getIPDetailsUrl(params.detailName)}${search && search[1] ? search[1] : ''}`,
+        href: `${getIPDetailsUrl(params.detailName, params.flowTarget)}${
```

Comment:
Since that we are adding `source/destination` to url path, we want to reflect that in breadcrumb as well?
i.g.
http://localhost:5601/app/siem#/network/ip/35.227.125.33/destination
breadcumb: SIEM / Network / 35.227.125.33 / destination

Reply:
@MichaelMarcialis what are your thoughts?

Reply:
Good question. I'd prefer we use the URL parameter. Personally, I don't believe we should add `Source` or `Destination` to the breadcrumbs for the following reasons:

- Doing so will give the user the impression that these are two separate pages. I'd rather convey the feeling that this is the same IP details page, just filtered by flow direction.

- If we were to add `Source` and `Destination` to the breadcrumbs, the IP address would theoretically become a parent page. Where would clicking the IP address in the breadcrumbs take the user? I don't believe there is a logical higher-level page to take the user at the moment.

Reply:
Completely agree with @MichaelMarcialis 👍 @angorayc ?

Reply:
Sounds good to me! Thanks a lot!